src/components/ProductList.js

```diff
@@ -0,0 +1,29 @@
+import ProductListItem from "./ProductListItem";
+
+
+const ProductList = ({arrayOfProducts, handleAddToCart}) => {
+
+    // console.log(arrayOfProducts);
+    // console.log(arrayOfInventory)
+    
+
+    return (
+
+        <ul className="productList wrapper">
+            {
+                arrayOfProducts.map((productObject) => {
```

Comment:
Likewise with above suggestion, I think you should be able to map the list items directly in the ProductListItem.js, then just display <ProductListItem /> in ProductList.js. Completely optional, but I think it would be a bit simpler since you would deal with less props and the listitem logic is kept in the list item component!

Reply:
Love the idea of simplifying this - it does feel very busy with the the props!